clients/src/test/java/org/apache/kafka/common/record/KafkaLZ4Test.java

```diff
@@ -64,7 +64,7 @@ public void testKafkaLZ4() throws IOException {
         ByteArrayOutputStream output = new ByteArrayOutputStream();
         KafkaLZ4BlockOutputStream lz4 = new KafkaLZ4BlockOutputStream(output, this.useBrokenFlagDescriptorChecksum);
         lz4.write(this.payload, 0, this.payload.length);
-        lz4.flush();
+        lz4.close();
```

Comment:
Also, it would be good to verify that `flush` still behaves as expected if called without `close`.